### ISSUE TITLE
Add color palette to Style Guide

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3623]
This color pallet was prepared after the November 2023 Book Dash to help with alignment of new scriberia illustrations to our existing design aesthetic. 

This needs to be added to the existing style guide in order to help with future design work.

![Color palette of TTW](https://github.com/the-turing-way/the-turing-way/assets/18509789/6597d751-71b9-4824-be17-10f4a7586006)

**Comment by nghuixin** [comment]
@aleesteele I'd love to work on this. But I'm not sure where this figure should be added to the Community Handbook; the most appropriate page I can think of is this [one on "Using figures in The Turing Way"](https://book.the-turing-way.org/community-handbook/style/style-figures). Or should be a completely separate page altogether?

**Comment by jyoti-bhogal** [comment]
@nghuixin and @aleesteele I am contributing to this issue during Book Dash November 2024. Please let me know if there is something specific that you would like me to contribute to.

**Comment by aleesteele** [comment]
@nghuixin - we can talk about this issue on Thursday :) 

@jyoti-bhogal - is it alright if we address this issue? I know you got pulled into another chapter during the book dash event!

**Comment by jyoti-bhogal** [comment]
@aleesteele yes, please go ahead with addressing this issue. Like you said, during the November 2024 Book Dash, I worked on another issue instead 🙂

**Comment by jyoti-bhogal** [comment]
I'd like to work on this issue during Book Dash May 2026.
